web/handlers.go

```diff
@@ -68,7 +68,9 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	c.SetSiteURLHeader(app.GetProtocol(r) + "://" + r.Host)
+	subpath, _ := utils.GetSubpathFromConfig(c.App.Config())
+	siteURLHeader := app.GetProtocol(r) + "://" + r.Host + subpath
```

Comment:
The returned `subpath` will always be either empty, or having a leading `/`, so this append will work as expected.

I've also checked downstream, and believe all instances of `siteURLHeader` via the context are used in ways that ought to use the subpath, vs. being used for a `Host` header or some such.